src/release/breaking-changes/material-3-migration.md

```diff
@@ -0,0 +1,515 @@
+
+Prior to the 3.16 release, you could opt in to the Material 3 changes by
+setting the `useMaterial3` flag to true. As of the Flutter 3.16 release
+(November 2023), `useMaterial3` is true by default.
+
+By the way, you _can_ recapture Material 2 behavior in your app by setting the
+`useMaterial3` to `false`. However, this is just a temporary solution. The
+`useMaterial3` flag _and_ the Material 2 implementation will eventually be
+removed as part of Flutter’s deprecation policy.
+
+### Colors
+
+The default values for `ThemeData.colorScheme` are updated to match
+the Material 3 Design spec.
+
+The `ColorScheme.fromSeed` constructor generates a `ColorScheme`
+derived from the given `seedColor`. The colors generated by this
+constructor are designed to work well together and meet contrast
+requirements for accessibility in the Material 3 Design system.
+
+When updating to the 3.16 release, your UI might look a little strange
+without the correct `ColorScheme`. To fix this, migrate to the
+`ColorScheme` generated from the `ColorScheme.fromSeed` constructor.
+
+Code before migration:
+
+```dart
+theme: ThemeData(
+  colorScheme: ColorScheme.light(primary: Colors.blue),
+),
+```
+
+Code after migration:
+
+```dart
+theme: ThemeData(
+  colorScheme: ColorScheme.fromSeed(seedColor: Colors.blue),
+),
+```
+
+To generate a content-based dynamic color scheme, use the
+`ColorScheme.fromImageProvider` static method. For an example of generating a
+color scheme, check out the [`ColorScheme` from a network image][] sample.
+
+[`ColorScheme` from a network image]: {{site.api}}/flutter/material/ColorScheme/fromImageProvider.html
+
+Changes to Flutter Material 3 include a new background color.
+`ColorScheme.surfaceTint` indicates an elevated widget.
+Some widgets use different colors.
+
+To return your app’s UI to its previous behavior (which we don't particularly
+recommend):
+* Set `Colors.grey[50]!` to `ColorScheme.background`
+   (when the theme is `Brightness.light`).
+* Set  `Colors.grey[850]!`to `ColorScheme.background`
+   (when the theme is `Brightness.dark`).
+
+Code before migration:
+
+```dart
+theme: ThemeData(
+  colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
+),
+```
+
+Code after migration:
+
+```dart
+theme: ThemeData(
+  colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple).copyWith(
+    background: Colors.grey[50]!,
+  ),
+),
+```
+
+```dart
+darkTheme: ThemeData(
+  colorScheme: ColorScheme.fromSeed(
+    seedColor: Colors.deepPurple,
+    brightness: Brightness.dark,
+  ).copyWith(background: Colors.grey[850]!),
+),
+```
+
+The `ColorScheme.surfaceTint` value indicates a component's elevation in
+Material 3. Some widgets might use both `surfaceTint` and `shadowColor` to
+indicate elevation (for example, `Card` and `ElevatedButton`) and others might
+only use `surfaceTint` to indicate elevation (such as `AppBar`).
+
+If you must return to a widget’s previous behavior, set, set `Colors.transparent`
```

Comment:
Suggest: 'To return to the widget's previous behavior'

Reply:
The reason I said "must" is that this page spends a lot of time telling folks how to return to previous behavior, but we really don't want to encourage that.